.github/actions/add-label-artifact/action.yaml

```diff
@@ -6,7 +6,7 @@ inputs:
     description: Name
     required: true
   value:
-    description: Value ("true" or "false")
+    description: Value ("true" or "add" to add the label, "false" or "remove" to remove the label)
```

Comment:
"add/remove" is clearer than "true/false", but to avoid breaking existing PRs, support both until we can update all callers and wait a few weeks for branches to update